lib/OpenQA/Test/PatchDeparse.pm

```diff
@@ -1,4 +1,6 @@
 package OpenQA::Test::PatchDeparse;
+use strict;
+use warnings 'all';
```

Comment:
Yes, but that's because perlcritic doesn't know that Test::Most uses strict and warnings automatically.
I don't know how to address that currently. But we don't want to add those two lines again everywhere I would say.

Reply:
Found a nice way :) Added the module to the `.perlcriticrc` config and now it "knows" that `Test::most` brings strict & warnings.